Documentation/devicetree/bindings/vendor-prefixes.yaml

```diff
@@ -319,6 +319,8 @@ patternProperties:
     description: Elgin S/A.
   "^elida,.*":
     description: Shenzhen Elida Technology Co., Ltd.
+  "^embeddedts,.*":
+    description: embeddedTS
```

Comment:
This was my mistake from commit cab0cd3692159c1c9fa224dbb111f758ae54946e where I renamed it from technologic to embeddedts. I can go back and touch that in our kernel branches, I had forgotten to add that to my to do list. I'd say revert this commit now (put embeddedts back in the "wrong" alphabetical order) and once this is merged I can go revert the vendor prefixes so its all back to technologic.